django_app/redbox_app/jinja2.py

```diff
@@ -71,6 +72,7 @@ def environment(**options):
     env = jinja2.Environment(  # nosec: B701 # noqa: S701
         **{
             "autoescape": True,
+            "extensions": [CompressorExtension],
```

Comment:
We found issues with using compressor on certain peoples laptops. Might be worth looking into alternatives if possible, @KevinEtchells knows more about this part than me

Reply:
We're using https://parceljs.org/ for bundling JS and CSS, which can also do compression